.github/workflows/ci.yml

```diff
@@ -256,7 +258,9 @@ jobs:
         uses: ./.github/actions/load-docker-images
 
       - name: Launch external services
-        run: yarn workspace @apps/hash-external-services deploy up --detach
+        run: |
+          yarn turbo codegen --filter '@apps/hash-external-services'
+          yarn workspace @apps/hash-external-services deploy up --detach
```

Comment:
Is this `codegen` step required because we're mounting the python files? Could we remove the need to explicitly call codgen by having a `codegen` dependency from the `deploy` task?

Reply:
No, it will generate `.env.local` 😄

Reply:
I see, that makes sense, but a little strange :)

Reply:
Yeah, but it was the only way I found to avoid building the images. Locally, we want the dependencies (and they should be cached in most cases), but in CI we have to work around this. 🙁